docs/source/guides/network_node/staking_guide.rst

```diff
@@ -385,6 +387,23 @@ Wind down is *disabled* by default. To start winding down an existing stake:
     (nucypher)$ nucypher stake winddown --hw-wallet
 
 
+Snapshots
+*********
+
+Taking snapshots is *enabled* by default. Snapshots must be enabled to participate in the DAO. To stop taking snapshots:
```

Comment:
Maybe mention why a staker may be interested in disabling snapshots, which is basically to have a slight reduction on gas expenditure.

Reply:
```suggestion
Taking snapshots is *enabled* by default. Snapshots must be enabled to participate in the DAO, but it has a slight cost in gas every time your staking balance changes. To stop taking snapshots:
```

Reply:
Ok, I'll add today but later